Cargo.toml

```diff
@@ -47,6 +47,7 @@ include.workspace = true
 features = ["unstable-doc"]
 rustdoc-args = ["--cfg", "docsrs"]
 cargo-args = ["-Zunstable-options", "-Zrustdoc-scrape-examples"]
+rustdoc-args = ["--generate-link-to-definition"]
```

Comment:
In case you didn't see, we already have `rustdoc-args`, so it needs to be appended to that instead

Reply:
... I wonder how I missed that. Sorry about that. Updating shortly.